libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -433,6 +445,9 @@ void elasticsearch_plugin::plugin_initialize(const boost::program_options::varia
    if (options.count("elasticsearch-index-prefix")) {
       my->_elasticsearch_index_prefix = options["elasticsearch-index-prefix"].as<std::string>();
    }
+   if (options.count("elasticsearch-operation-object")) {
```

Comment:
Do you really want to have this optional? IMO this is much better, keeping the old stringified version doesn't make much sense.

Reply:
Only worried about it if it delays too much the replay and to consider applications that may be using the string in `op` already. Agree this is much better, having it enabled by default can be another option. I am open also to have this non optional and doing it by default.